environmentlogger/environmental_logger_json2netcdf.py

```diff
@@ -90,6 +90,12 @@
                     u'us': {"original":"microsecond", "SI":"second", "power":1e-6},
                     u'ppm': {"original":"pascal meter-2", "SI":"pascal meter-2", "power":1}, 
```

Comment:
Yes this is ppm. For CO2 the standard name is `mole_fraction_of_carbon_dioxide_in_air`	and units can be `umol/mol` which is equivalent to ppm.